src/os/ui/draw/basedrawcontrols.js

```diff
@@ -9,6 +9,8 @@ goog.require('os.config.Settings');
 goog.require('os.data.RecordField');
 goog.require('os.interaction.DragZoom');
 goog.require('os.metrics.Metrics');
+goog.require('os.ogc.OGCService');
```

Comment:
Requires for this in these files don't look to be needed anymore. I think you can get rid of the `declareLegacyNamespace` on that module too if you take these out.